src/page-response-impact/responseChartData.tsx

```diff
@@ -22,9 +22,8 @@ const NUM_DAYS = 90;
 const NUM_SEIR_CATEGORIES = 9;
 
 export type SystemWideData = {
+  [key: string]: number;
```

Comment:
Should we make this type more specific and use the properties defined in the EpidemicModelContext?

Reply:
That's what I had originally but Typescript would then not allow me to access the object via the key on line 97. I'm not sure the best way to approach this, I'm still trying to figure out Typescript.

Reply:
I suppose I could avoid the Object.keys iteration and just explicitely sum each of the attributes in the `SystemWideData`. That might be easier to read anyway. Let me give that a try to see what it looks like.

Reply:
Also, you could use the `Partial` utility like this:
```
export type SystemWideData = Partial<ModelInputsPersistent> & {
  hospitalBeds: number;
  incarceratedPopulation: number;
}

```

Reply:
Cool! Good tip. I ended up using the `Partial` utility and it seems to be the right solution. I also left the other attributes in the declaration to make it clear what is being summed at the system level. Ready for another look.